examples/XGBoost_d4p_prediction.ipynb

```diff
@@ -0,0 +1,453 @@
+    "y_test = readcsv(test_file, range(3, 4), t=np.float32)\n",
+    "\n",
+    "# Datasets creation\n",
+    "xgb_train = xgb.DMatrix(X_train, label=np.array(y_train))\n",
+    "xgb_test = xgb.DMatrix(X_test, label=np.array(y_test))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "unnecessary-librarian",
+   "metadata": {},
+   "source": [
+    "## Training the Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "indoor-allocation",
+   "metadata": {},
+   "source": [
+    "**Fitting and training the model** using the training dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "sufficient-google",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# training parameters setting\n",
+    "params = {\n",
+    "    'max_bin': 256,\n",
+    "    'scale_pos_weight': 2,\n",
+    "    'lambda_l2': 1,\n",
+    "    'alpha': 0.9,\n",
+    "    'max_depth': 8,\n",
+    "    'num_leaves': 2**8,\n",
+    "    'verbosity': 0,\n",
+    "    'objective': 'multi:softmax',\n",
+    "    'learning_rate': 0.3,\n",
+    "    'num_class': 5,\n",
+    "}\n",
+    "\n",
+    "# Training\n",
+    "xgb_model = xgb.train(params, xgb_train, num_boost_round=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "several-darkness",
+   "metadata": {},
+   "source": [
+    "## Using daal4py for Prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "clear-gilbert",
+   "metadata": {},
+   "source": [
+    "### XGBoost Prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "molecular-indicator",
+   "metadata": {},
+   "source": [
+    "We will also make a prediction using XGBoost for accuracy/performance comparison."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "worldwide-inspiration",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# XGBoost prediction (for accuracy comparison)\n",
+    "t0 = time.time()\n",
+    "xgb_prediction = xgb_model.predict(xgb_test)\n",
+    "t1 = time.time()\n",
+    "xgb_errors_count = np.count_nonzero(xgb_prediction - np.ravel(y_test))\n",
+    "\n",
+    "xgb_total = t1-t0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "separated-connection",
+   "metadata": {},
+   "source": [
+    "### daal4py Prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bottom-recipe",
+   "metadata": {},
+   "source": [
+    "Now let's make a prediction using daal4py for increased performance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "graphic-finish",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Conversion to daal4py\n",
+    "daal_model = d4p.get_gbt_model_from_xgboost(xgb_model)\n",
+    "\n",
+    "# daal4py prediction\n",
+    "daal_predict_algo = d4p.gbt_classification_prediction(\n",
+    "    nClasses=params[\"num_class\"],\n",
+    "    resultsToEvaluate=\"computeClassLabels\",\n",
```

Comment:
minor comment: It would be good to show that not only class labels are the same, but also calculated probability is close to XGBoost (via log_loss metric as an example) by providing computeClassLabels|computeClassProbabilities.